plugin/pxr/maya/lib/usdMaya/testenv/testUsdMayaDiagnosticDelegate.py

```diff
@@ -144,10 +147,13 @@ def testBatching_DelegateRemoved(self):
 
             cmds.unloadPlugin('pxrUsd', force=True)
 
-            for i in xrange(5):
+            for i in range(5):
                 Tf.Status("no delegate, this will be lost %d" % i)
         log = self._StopRecording()
 
+        # assertCountEqual in python 3 is equivalent to assertItemsEqual
+        if sys.version_info[0] >= 3:
+            self.assertItemsEqual = self.assertCountEqual
```

Comment:
Maybe do this in `setUpClass()` to avoid repeating it?

Reply:
See e594505ae0d7ba2258f47edfb48bad0b558c2424